docs/chapters/baseplatform/deploying-and-booting-the-image.rst

```diff
@@ -0,0 +1,64 @@
+Writing the image
+=================
+
+This section describes how to write the build output, a ready disk-image, to a removable media. Assuming you are positioned in the PELUX top-level Yocto directory follow the instructions stated below. Further information can be found in the `Poky documentation`_.
+
+NUC / Minnowboard
+-----------------
+
+There are three different methods of writing an Intel ``.hddimg`` or ``.wic`` image to a USB-stick.
+
+1. Use the ``dd`` utility to write the image to the raw block device. For example:
+
+.. code-block:: bash
+
+    dd if=build/tmp/deploy/images/intel-corei7-64/core-image-pelux-intel-corei7-64.hddimg \
+    of=<host-device> bs=4M
+
+2. Poky provides a convenience script for writing images. Using an unmounted USB-stick or SD-card, do:
```

Comment:
As you mentioned the script will be depricated, then I think we can remove the references to it.